Packs/Endace/Playbooks/playbook_Endace_Search_Archive_Download.yml

```diff
@@ -0,0 +1,606 @@
+    nexttasks:
+      '#none#':
+      - '25'
+    scriptarguments:
+      filename:
+        simple: ${Endace.Archive.Task.FileName}
+      filesizelimit: {}
+    separatecontext: false
+    view: "{\n  \"position\": {\n    \"x\": 655,\n    \"y\": 2120\n  }\n}"
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+  '36':
+    id: '36'
+    taskid: ae66399d-5356-4576-81d4-bababf3f564d
+    type: regular
+    task:
+      id: ae66399d-5356-4576-81d4-bababf3f564d
+      version: -1
+      name: endace-create-archive
+      description: Create an archive task to archive packets of interest on EndaceProbe.
+        Archived packets can later be downloaded from EndaceProbe as a PCAP file.
+        Archived Files never expire. Allowed chars are text, numbers, dash and underscore.
+      script: Endace|||endace-create-archive
+      type: regular
+      iscommand: true
+      brand: Endace
+    nexttasks:
+      '#none#':
+      - '24'
+    scriptarguments:
+      archive_filename:
+        simple: pcaparchive
+      dest_host_list:
+        simple: ${inputs.dest_host_list}
+      dest_port_list:
+        simple: ${inputs.dest_port_list}
+      end:
+        simple: ${inputs.end}
+      protocol:
+        simple: ${inputs.protocol}
+      src_host_list:
+        simple: ${inputs.src_host_list}
+      src_port_list:
+        simple: ${inputs.src_port_list}
+      start:
+        simple: ${inputs.start}
+      timeframe:
+        simple: ${inputs.timeframe}
+    separatecontext: false
+    view: "{\n  \"position\": {\n    \"x\": 275,\n    \"y\": 1070\n  }\n}"
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+view: "{\n  \"linkLabelsPosition\": {\n    \"14_7_no\": 0.32,\n    \"26_7_no\": 0.1,\n\
+  \    \"27_7_no\": 0.25,\n    \"9_26_yes\": 0.67,\n    \"9_7_no\": 0.1\n  },\n  \"\
+  paper\": {\n    \"dimensions\": {\n      \"height\": 2485,\n      \"width\": 985,\n\
+  \      \"x\": 50,\n      \"y\": 50\n    }\n  }\n}"
+inputs:
+- key: start
+  value: {}
+  required: false
+  description: Event Start Time in ISO 8601 format
+  playbookInputQuery:
+- key: end
+  value: {}
+  required: false
+  description: Event End Time in ISO 8601 format
+  playbookInputQuery:
+- key: src_host_list
+  value: {}
+  required: false
+  description: List of Source IP addresses to search
+  playbookInputQuery:
+- key: dest_host_list
+  value: {}
+  required: false
+  description: List of Destination IP addresses to search
+  playbookInputQuery:
+- key: src_port_list
+  value: {}
+  required: false
+  description: List of Source Port addresses to search
+  playbookInputQuery:
+- key: dest_port_list
+  value: {}
+  required: false
+  description: List of Destination Port addresses to search
+  playbookInputQuery:
+- key: protocol
+  value:
+    simple: TCP
+  required: false
+  description: TCP or UDP
+  playbookInputQuery:
+- key: timeframe
```

Comment:
This should support more than just seconds
The user should be able to input values such as last 1 hour, last 1 day, last 1 week etc

Reply:
This is how it has been designed based on our use case. Changing this to accommodate at this stage would need lot of rework without much of an added benefit other than UX .

Reply:
@michalgold sounds like something that will be very helpful for hunting